src/Components/CartCard/CartCard.js

```diff
@@ -47,7 +49,15 @@ export default function CartCard({ cartList }) {
                   >
                     Remove from Cart
                   </button>
-                  <button className="horizontal-btn moveItem-btn">
+                  <button
+                    className="horizontal-btn moveItem-btn"
+                    onClick={() =>
+                      wishlistDispatch({
+                        type: "ADD_TO_WISHLIST",
```

Comment:
this type values can be moved to a separate file and it can be referred in any number of places

Reply:
the shop now button is showing a box below it.